src/AppInstallerCommonCore/ExperimentalFeature.cpp

```diff
@@ -79,10 +75,6 @@ namespace AppInstaller::Settings
             return ExperimentalFeature{ "Direct MSI Installation", "directMSI", "https://aka.ms/winget-settings", Feature::DirectMSI };
         case Feature::Resume:
             return ExperimentalFeature{ "Resume", "resume", "https://aka.ms/winget-settings", Feature::Resume };
-        case Feature::ConfigurationDSCv3:
-            return ExperimentalFeature{ "Support for DSC v3", "dsc3", "https://aka.ms/winget-settings", Feature::ConfigurationDSCv3 };
```

Comment:
Have we, or will we ever used a link that wasn't `https://aka.ms/winget-settings`?